public/Restore-DbaDatabase.ps1

```diff
@@ -650,7 +650,8 @@ function Restore-DbaDatabase {
                     Write-Message -Message "$Database does not exist on $RestoreInstance" -level Warning
                     continue
                 }
-                if ($RestoreInstance.Databases[$Database].Status -ne "Restoring") {
+
+                if (@("Restoring", "Normal, Standby") -notcontains $RestoreInstance.Databases[$Database].Status) {
                     Write-Message -Message "$Database on $RestoreInstance is not in a Restoring State" -Level Warning
```

Comment:
the message here is not related anymore to the "Restoring" state, so it needs to convey the reason why it's skipped

Reply:
Good catch. I'll update it.

Reply:
Done.